python/cac_tripplanner/destinations/views.py

```diff
@@ -47,10 +47,10 @@ def home(request):
     }
     if request.GET.get('destination') is not None:
         # If there's a destination in the URL, go right to directions
-        context['tab'] = 'map'
+        context['body_class'] = 'body-map body-map-dir body-has-sidebar-banner'
```

Comment:
5139467 changes the view back to just picking a tab name and moves the logic for setting the initial body classes into `base.html`.  Still not ideal, but so much of the styling hangs on body class that it's not clear to me how to move it further down in the templates or purely into javascript.

Also note that using both `.body-map` and `.body-map-{tab}` is convenient since there's a lot of styling that's common to both.  `.body-map` without one of the subclasses isn't actually a valid thing to do.  Maybe there's an easy way to switch the styles to not need the shared class, but if not it doesn't seem worth the trouble.